priv/templates/phx.gen.auth/confirmation_controller.ex

```diff
@@ -26,16 +26,16 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
   end
 
   # Do not log in the <%= schema.singular %> after confirmation to avoid a
-  # leaked token giving the <%= schema.singular %> access to the account.
+  # leaked token giving the <%= schema.singular %> access to the <%= schema.human_singular %>.
```

Comment:
Don't think these should actually be updated, should be left as account?

Reply:
I agree. Thanks for switching these back. :+1: